scenario_player/main.py

```diff
@@ -323,6 +328,9 @@ def run_(
             )
         notify_tasks_callable = post_task_state_to_rc
 
+    # TODO: make this used
+    print(notify_tasks_callable)
```

Comment:
is this intended to stay?

Reply:
It was passed to `orchestrate` before, but never used afterwards. Maybe @ulope can shine some light on this?

Reply:
The idea of that callback was to have a way of notifying external tools (e.g. rocketchat) about the task state. It's very possible that this got broken in one of the refactorings along the way.

Reply:
Ok, then I create an issue to bring it back when we need it